gui/default/syncthing/app.js

```diff
@@ -26,6 +26,7 @@ syncthing.config(function ($httpProvider, $translateProvider, LocaleServiceProvi
         prefix: 'assets/lang/lang-',
         suffix: '.json'
     });
+    $translateProvider.fallbackLanguage('en');
```

Comment:
Oh right, this snuck in too. See also some previous discussion here: https://github.com/syncthing/syncthing/commit/1c5e4ba8adf13162f219a318f5012f3fe41a13fe#r130790654